src/microbe_stage/BackgroundPlane.tscn

```diff
@@ -0,0 +1,117 @@
+
+[sub_resource type="FastNoiseLite" id="FastNoiseLite_dor76"]
+frequency = 0.0035
+
+[sub_resource type="NoiseTexture2D" id="NoiseTexture2D_f6pop"]
+seamless = true
+noise = SubResource("FastNoiseLite_dor76")
+
+[sub_resource type="FastNoiseLite" id="FastNoiseLite_ve355"]
+seed = 2
+frequency = 0.0017
+
+[sub_resource type="NoiseTexture2D" id="NoiseTexture2D_tsmw6"]
+seamless = true
+noise = SubResource("FastNoiseLite_ve355")
+
+[sub_resource type="ShaderMaterial" id="2"]
+resource_local_to_scene = true
+render_priority = -100
+shader = ExtResource("4_5y6kw")
+shader_parameter/lightLevel = 1.0
+shader_parameter/repeats = Vector2(2, 1)
+shader_parameter/distortionFactor = 0.0
+shader_parameter/distortionSpeed = 0.0025
+shader_parameter/layerAnimateSpeed = 0.4
+shader_parameter/worldPos = Vector2(0, 0)
+shader_parameter/layer0 = ExtResource("5_waqnb")
+shader_parameter/layer1 = ExtResource("6_qx2cp")
+shader_parameter/layer2 = ExtResource("7_g3bc7")
+shader_parameter/layer3 = ExtResource("8_yxu5g")
+shader_parameter/noiseTex1 = SubResource("NoiseTexture2D_f6pop")
+shader_parameter/noiseTex2 = SubResource("NoiseTexture2D_tsmw6")
+
+[sub_resource type="ViewportTexture" id="ViewportTexture_duokn"]
+viewport_path = NodePath("SubViewport2")
+
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_1sr76"]
+resource_local_to_scene = true
+render_priority = -100
+shader = ExtResource("1_iyjvf")
+shader_parameter/blurStep = Vector2(0, 1)
+shader_parameter/blurAmount = 1.0
+shader_parameter/textureAlbedo = SubResource("ViewportTexture_duokn")
+
+[sub_resource type="ViewportTexture" id="ViewportTexture_vbioy"]
+viewport_path = NodePath("SubViewport")
+
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_tdrj5"]
+resource_local_to_scene = true
+shader = ExtResource("3_s3lms")
+shader_parameter/blurStep = Vector2(1, 0)
+shader_parameter/blurAmount = 1.0
+shader_parameter/textureAlbedo = SubResource("ViewportTexture_vbioy")
+
+[node name="BackgroundPlane" type="Node3D" node_paths=PackedStringArray("backgroundPlane", "blurPlane", "blurColorRect", "subViewport1", "subViewport2")]
+script = ExtResource("2_eprmh")
+backgroundPlane = NodePath("BackgroundPlane")
+blurPlane = NodePath("BlurPlane")
+blurColorRect = NodePath("SubViewport2/ColorRect")
+subViewport1 = NodePath("SubViewport")
+subViewport2 = NodePath("SubViewport2")
+
+[node name="BackgroundPlane" type="CSGMesh3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, -4.37114e-08, -1, 0, 1, -4.37114e-08, 0, 0, -15)
+cast_shadow = 0
+mesh = SubResource("1")
+material = SubResource("2")
+
+[node name="BlurPlane" type="CSGMesh3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, -1.62921e-07, -1, 0, 1, -1.62921e-07, 0, 0, -20)
+visible = false
+cast_shadow = 0
+mesh = SubResource("1")
+material = SubResource("ShaderMaterial_1sr76")
+
+[node name="SubViewport2" type="SubViewport" parent="."]
+disable_3d = true
+handle_input_locally = false
+size = Vector2i(1280, 720)
+render_target_clear_mode = 1
```

Comment:
This is still the wrong property, this is `Clear Mode` whereas the right thing is to set `Update Mode` to `Disabled`

So in this text view it should be `render_target_update_mode = 0` (and this clear mode should be left to default).